libs/product/src/facades/product/product.facade.ts

```diff
@@ -33,23 +32,43 @@ export class DaffProductFacade<T extends DaffProduct = DaffProduct> implements D
     this.loading$ = this.store.pipe(select(this.selectors.selectSelectedProductLoadingState));
 		this.product$ = this.store.pipe(select(this.selectors.selectSelectedProduct));
 	}
-	
+
+	/**
+	 * Get a product.
+	 * @param id a product id
+	 */
 	getProduct(id: string): Observable<T> {
 		return this.store.pipe(select(this.selectors.selectProduct, { id }));
 	}
 
+	/**
+	 * Whether a particular product has a discount.
+	 * @param id a product id
+	 */
 	hasDiscount(id: string): Observable<boolean> {
 		return this.store.pipe(select(this.selectors.selectProductHasDiscount, { id }));
 	}
 
+	/**
+	 * Get the discount amount of a product.
+	 * @param id a product id
+	 */
 	getDiscountAmount(id: string): Observable<number> {
 		return this.store.pipe(select(this.selectors.selectProductDiscountAmount, { id }));
 	}
 
+	/**
+	 * Get the discount percent of a product
```

Comment:
period

Reply:
this comment was removed in favor of the interface